src/rdkafka_queue.c

```diff
@@ -517,7 +517,35 @@ int rd_kafka_q_serve (rd_kafka_q_t *rkq, int timeout_ms,
 }
 
 
+int invalid_all_old_msgs_in_array(rd_kafka_op_t **rkos,
+                                  int32_t version,
+                                  rd_kafka_message_t **rkmessages,
+                                  int cnt,
+                                  size_t rkmessages_size) {
+    TAILQ_HEAD(, rd_kafka_op_s) tmpq = TAILQ_HEAD_INITIALIZER(tmpq);
+    rd_kafka_op_t *rko, *next;
+    rd_kafka_message_t *tmp[rkmessages_size];
```

Comment:
Variable array sizes is not compatible with older compilers, and is also not stack-size safe, use rd_malloc.